lib/xform.js

```
@@ -41,18 +55,159 @@ var XFormState = State.extend(function (self, name, opts) {
 
     // Defaults
     opts = _.defaults(opts || {}, {
-        next:null,
+        next: null,
+        xforms_service_url: 'prd-vumi-xforms.aws.prk-host.net',
+        xforms_service_error_message: 'Error contacting the xforms service',
     });
 
     State.call(self, name, opts);
 
     // Store options
     self.next = opts.next;
+    self.xforms_service_url = opts.xforms_service_url;
+    self.xforms_service_error_message = opts.xforms_service_error_message;
+    self.xform = opts.xform;
```

Should we raise an error if ``opts.xform`` is undefined?